.gitignore

```diff
@@ -6,3 +6,4 @@ spec/dummy/db/*.sqlite3
 spec/dummy/log/*.log
 spec/dummy/tmp/
 spec/dummy/.sass-cache
+.idea
```

Comment:
@andie-chernysh, it might be a good '.idea' (hehe) to add this line to your global gitignore.  I'm guessing that those .idea files are your brainstorming/notes files and since other developers don't necessarily use them, we don't need these lines in the project .gitignores.  I don't actually mind them being in the project files though.